### ISSUE TITLE
Add ability to edit "BPM" of a song

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#1500]
_From [janoszh100](https://code.google.com/u/106872840392257424992/) on February 23, 2011 21:52:02_

Please describe the feature you're requesting in as much detail as possible. Ability to manually edit "BPM" of a song.

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=1500_

**Comment by pedrobaeza** [comment]
It would be great indeed to have automatic BPM recognition integrated.

**Comment by palomnik** [comment]
I agree, this is an ESSENTIAL feature.  Better yet, create a tag space for both BPM and Key

**Comment by johantiden** [comment]
Indeed, just editing the tag should be simple to implement.

**Comment by NullVoxPopuli** [comment]
I just ran in to the need for this 😅

**Comment by AlexandreBeckao** [comment]
I've been using Clementine for many years, and I still hope for the day when it will just read BPM correctly.
I think it's about time to implement this feature, it would be quite an achievement for the player and its loyal community...
Still, thank you very much for keeping it running, despite the hardships...

**Comment by 64kramsystem** [comment]
> I've been using Clementine for many years, and I still hope for the day when it will just read BPM correctly. I think it's about time to implement this feature, it would be quite an achievement for the player and its loyal community... Still, thank you very much for keeping it running, despite the hardships...

I've solved the problem by setting the BPM in the `Comment` field.
